content/docs/command-reference/add.md

```diff
@@ -70,6 +73,26 @@ large files. DVC also supports other link types for use on file systems without
 `reflink` support, but they have to be specified manually. Refer to the
 `cache.type` config option in `dvc config cache` for more information.
 
+### Transferring data directly to the remote
+
+When you have a very big dataset that you want to move from some remote location
+to one of your remotes, but at the same time you don't have time or resources to
+store it locally on your local system, you can use `--to-remote` to add that
+remote location straight to remote instead of your local workspace. The remote
+location can be any of the ones that are listed under
+[import-url](/doc/command-reference/import-url) page. When you add a remote
+location with `--to-remote`, it will get the dataset from the given location and
+transfer it to the remote you specified (or the default one). It will create a
+DVC file just like you added something locally, but there won't be any data that
+you can access, unless you [pull](/doc/command-reference/pull) it. In that case,
+it will pull it from the remote storage unit to your workspace and you can start
+using it.
```

Comment:
Copy edit:
```suggestion
When you have a very big dataset that you want to move from some external location
to [remote storage](/doc/command-reference/remote) while avoiding storing it locally,
you can use the `--to-remote` option. This will transfer the target data
to a remote of your choice (or the default one). A `.dvc` file will be created normally,
but the data won't be found in your local project until you `dvc pull` it.
```
> Needs formatting.